scss/bitstyles/utilities/col-span/col-span.stories.mdx

```diff
@@ -0,0 +1,110 @@
+        <li class="u-bg--background u-padding-m a-card">Grid child 5</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 6</li>
+      </ul>
+    `}
+  </Story>
+</Canvas>
+
+# Responsive col-span
+
+The `col-span` classes are also all available at `s`, `m`, and `xl` breakpoints.
+
+<Canvas isColumn>
+  <Story name="u-col-span-6@s">
+    {`
+      <ul class="u-grid u-gap-m u-grid-cols-6 u-padding-m u-bg--gray-10 a-list-reset">
+        <li class="u-bg--brand-2 u-padding-m a-card u-col-span-6@s">Grid child 1, span 6 @s</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 2</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 3</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 4</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 5</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 6</li>
+      </ul>
+    `}
+  </Story>
+  <Story name="u-col-span-6@m">
+    {`
+      <ul class="u-grid u-gap-m u-grid-cols-6 u-padding-m u-bg--gray-10 a-list-reset">
+        <li class="u-bg--brand-2 u-padding-m a-card u-col-span-6@m">Grid child 1, span 6 @m</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 2</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 3</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 4</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 5</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 6</li>
+      </ul>
+    `}
+  </Story>
+  <Story name="u-col-span-6@l">
+    {`
+      <ul class="u-grid u-gap-m u-grid-cols-6 u-padding-m u-bg--gray-10 a-list-reset">
+        <li class="u-bg--brand-2 u-padding-m a-card u-col-span-6@l">Grid child 1, span 6 @l</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 2</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 3</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 4</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 5</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 6</li>
+      </ul>
+    `}
+  </Story>
+  <Story name="u-col-span-6@xl">
+    {`
+      <ul class="u-grid u-gap-m u-grid-cols-6 u-padding-m u-bg--gray-10 a-list-reset">
+        <li class="u-bg--brand-2 u-padding-m a-card u-col-span-6@xl">Grid child 1, span 6 @xl</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 2</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 3</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 4</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 5</li>
+        <li class="u-bg--background u-padding-m a-card">Grid child 6</li>
+      </ul>
+    `}
+  </Story>
+</Canvas>
+
+## Customization
+
+The available column-spans can be customized by overriding the `$bitstyles-col-span` Sass variable:
```

Comment:
Just outta curiosity: what would be a use case for overriding those? Make a more "fine grained" grid with e.g. 32 columns...?

Reply:
Yes exactly — this is just 6 for simplicity, and because we normally need more in admin UIs, but 12 or 16-column grids are common (32 is maybe overkill :D)